make_profiler/report/script.js

```diff
@@ -50,6 +50,7 @@ async function getStatus(url) {
         }
         throw new Error(errorTxt);
     }).then((responseJson) => {
+        errorTxt = "Something went wrong parsing report details"
```

Comment:
why error text is set in `then()` which is supposed to be expected execution path? could you move error handling in `catch()`?